docs/databases/redis/redis-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,189 @@
+		echo never > /sys/kernel/mm/transparent_hugepage/enabled
+	fi
+    ~~~
+
+
+You can confirm that it worked with command
+
+		cat /sys/kernel/mm/transparent_hugepage/enabled
+
+which should output `always madvise [never]`.
+
+## Distributed Redis
+
+Redis provides several options for setting up distributed data stores. The simplest one is the *master-slave replication*, which allows you to have a real-time copy (or multiple copies) of master server data. It will also allow to distribute reads among group of slave copies, as long as all write operations are handled by master server.
+
+Master-slave setup described above can be made highly available with *Redis Sentinel*. Sentinel will monitor both master and slave instances, and will perform automatic failover if master node is not working as expected. That means that one of the slave nodes will be elected master and all other slave nodes will be configured to use a new master.
+
+Starting from version 3.0, there is also a *Redis Cluster*, which is a data sharding solution with automatic management, handling failover and replication. With Redis Cluster you are able to automatically split your dataset among multiple nodes, which is useful when your dataset is larger than a single server RAM. It also gives you an ability to continue operations when a subset of the nodes are experiencing failures or are unable to communicate with the rest of the cluster.
+
+In this guide you will set up master slave replication as it is the simplest option.
+
+
+## Setting Up Master Slave Replication
+
+1.  Setup Linode with slave Redis instance using **Redis Installation** and **Redis Configuration** steps from this guide.
+
+2.  Configure [Private IP Addresses](/docs/networking/remote-access#adding-private-ip-addresses) on both Linodes, and make sure you can access master Linode's private IP address from slave. You will use only private addresses for replication traffic for security reasons.
+
+3.  Configure master Redis instance to listen on private IP address by updating `bind` configuration option in `redis.conf`
+
+{: .file-excerpt }
+/etc/redis/redis.conf
+:   ~~~
+    bind 127.0.0.1 <master_private_ip>
+    ~~~
+
+
+Restart `redis-server` to apply the changes:
+
+	    sudo service redis-server restart
+
+4.  Configure slave instance by adding `slaveof` directive into `redis.conf` to setup the replication -
+
+{: .file-excerpt }
+/etc/redis/redis.conf
+:   ~~~
+    slaveof <master_private_ip> 6379
+    ~~~
+
+The `slaveof` directive takes two arguments: the first is the IP address of the master node; the second is the Redis port specified in the master's configuration.
+
+Restart the slave Redis instance:
+
+	    sudo service redis-server restart
+
+After restart slave instance will attempt to synchronize its data set to the master and then propagate the changes.
+
+5.  Test that the replication works. On master Linode, run `redis-cli` and execute command `set 'a' 1`
```

Comment:
@alexfornuto , @EdwardAngert can you please look into it one more time ? I've checked again and on master node everything works fine.